Automattic/wp-calypso
# Stats highlight cards - add update timeframe notice

Related to # Automattic/dotcom-forge#8781

## Proposed Changes

HOLD ON MERGING - for reduced cache for the daily highlights in D159517-code. Before this the new message is only true on annual stats and not daily.

* Adds an "Updates every 30 minutes" note to stats highlights card sliders headers.
* Adds a common component to be used as a shared wrapper for these headers, applies them to the weekly/monthly highlights cards and the annual highlights cards. The components used the same HTML elements and class names, but did not share/consolidate those via react components. 

BEFORE
<img width="1029" alt="Screenshot 2024-08-26 at 1 21 40 PM" src="https://github.com/user-attachments/assets/eec5429d-2a71-425f-9510-da30dcfaa4d7">
<img width="1027" alt="Screenshot 2024-08-26 at 1 21 52 PM" src="https://github.com/user-attachments/assets/16fb60e6-d500-4867-8c56-a78199b812d1">



AFTER

<img width="1035" alt="Screenshot 2024-08-26 at 1 22 01 PM" src="https://github.com/user-attachments/assets/26bec46f-3cdd-4949-a405-1f8bbc3f13ac">
<img width="1024" alt="Screenshot 2024-08-26 at 1 22 11 PM" src="https://github.com/user-attachments/assets/00fd5b06-8b4c-4ef7-974a-c84a84b0f1fb">



## Why are these changes being made?


* It feels broken if you look at these stats numbers but know you have a different amount of posts, comments, views, etc. No matter how much we reduce cache timers or bust cache, there are always race conditions and other items that could make these areas present slightly out of date information.
* Adding a small note about an update timer helps fight a feeling of brokenness, and makes stale values feel more acceptable.
* More context at pe7F0s-25n-p2

## Testing Instructions



* Go to stats in calypso and look at the traffic and insights tabs.
* Verify the cards slider headers have a sub header section mentioning "Updates every 30 minutes"

## Pre-merge Checklist
